src/lib/utils/jsonResponses/BulkFileDownloadRequest.ts

```diff
@@ -0,0 +1,16 @@
+import { FileHandleAssociation } from './FileHandleAssociation'
+
+export enum ZipFileFormat {
+  CommandLineCache = 'CommandLineCache', // This is the default format. Zip file format structure is: (%1000)//
```

Comment:
Should keys be capitalized?

Reply:
I'm open to change on this, but my thinking here was to keep consistency with the enum defined by the backend